integration/test_collection.py

```diff
@@ -1407,15 +1411,10 @@ def test_near_image_flat(
     collection = client.collection.create(
         CollectionConfig(
             name=name,
-            vectorizer=Vectorizer.IMG2VEC_NEURAL,
+            vectorizer_config=Img2VecNeuralConfig(image_fields=["imageProp"]),
```

Comment:
Why do we sometimes have a factory (for Contextionary and None) and sometimes an extra class? I think it would make more sense to unify that?

Proposal:
- `None`as "No vectorizer"
- Text2VecContextionaryConfig

Reply:
I kept both to indicate its experimental nature and we should definitely make a decision on which is preferable in the near future. I hope that we will get some good feedback on this aspect through this week's user testing

Reply:
ok